WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -519,8 +525,11 @@ public void onPostButtonClicked(int buttonType, PostModel post) {
                 break;
             case PostListButton.BUTTON_TRASH:
             case PostListButton.BUTTON_DELETE:
+                // TODO: Async: Update this behavior to handle pressing Delete/Trash while the post has uploading media
+                // Currently, the button is tappable while media are uploading, but nothing happens
+
                 // prevent deleting post while it's being uploaded
-                if (!PostUploadService.isPostUploading(post)) {
+                if (!PostUploadService.isPostUploadingOrQueued(post)) {
```

Comment:
What do you think about adding a `Toast` message here in the `else` condition? Saying something like `This post is either being uploaded or in the queue to upload and can't be trashed`